### ISSUE TITLE
mosquitto_ctrl is not using options file specified via -o parameter

### ISSUE THREAD

**Issue by Vitaljok** [eclipse/mosquitto#2471]
Hello!

Followed the documentation for the Dynamic Security plugin (https://mosquitto.org/documentation/dynamic-security/) and, perhaps, found a bug.

I have started local broker on custom port with enabled Dynamic security plugin.
Following command works as expected:
```
$ mosquitto_ctrl -h localhost -p 23456 -u admin -P asd dynsec getDefaultACLAccess

publishClientSend    : deny
publishClientReceive : allow
subscribe            : deny
unsubscribe          : allow
```

As described in the documentation
> ... mosquitto_ctrl can load an options file ... from a default location ... `$HOME/.config/mosquitto_ctrl`

If I create mentioned file with following content
```
-h localhost
-p 23456
-u admin
-P asd
```

then it also works as expected:
```
$ mosquitto_ctrl dynsec getDefaultACLAccess

publishClientSend    : deny
publishClientReceive : allow
subscribe            : deny
unsubscribe          : allow
```

---

The documentation also says 
> You may override this behaviour by manually specifying an options file with `-o <path to options file>`.

But the file is not loaded and default options are used instead (it tries to connect to the broker on the default port).
```
$ mosquitto_ctrl -o /path/to/my/options dynsec getDefaultACLAccess

Error: Connection refused
```

Is this a bug or an undocumented feature?
Thanks!

Versions:
- mosquitto_ctrl version 2.0.14
- Ubuntu 18.04.6 LTS

**Comment by rillbert** [comment]
I believe I see the same behavior as described by @Vitaljok. 

Further on, the version of the docs that I follow (at https://mosquitto.org/documentation/dynamic-security/#installation) claims that the default config file name is `$HOME/.config/mosquitto_ctrl.conf` which has an extra `.conf` suffix compared to the name reported by @Vitaljok 

The only way I got to work was to not use the `-o` option and remove the `.config` suffix of the config file.

- mosquitto_ctrl version 2.0.14
- ubuntu 20.04

Cheers

**Comment by ralight** [comment]
Thank you both, I've fixed the two issues ready for 2.0.15.
